grant-project/zcash-grant-system
# Modal Sharing Links

Closes https://github.com/grant-project/zcash-grant-system/issues/190

##### Note: Based on https://github.com/grant-project/zcash-grant-system/pull/214, change `base` branch to develop after #214 is merged

### What this does

1. Moves `CopyInput` into it's own component file
2. Replaces Social Pop-up with Modal (so as not to directly link out to big social)

### How to test
1. Create a LIVE proposal (`flask create-proposals 1`)
2. Click on all social sharing links
3. Confirm Modal pops up with copyable input
4. Confirm copyable input is valid

##### Before
<img width="1228" alt="screen shot 2019-02-17 at 4 00 47 pm" src="https://user-images.githubusercontent.com/7861465/52919940-417e6e00-32cd-11e9-9979-6414e1938690.png">


##### After
<img width="1236" alt="screen shot 2019-02-17 at 4 00 31 pm" src="https://user-images.githubusercontent.com/7861465/52919942-43e0c800-32cd-11e9-9a0a-0acc28be9458.png">
